src/ape_accounts/_cli.py

```diff
@@ -64,25 +64,39 @@ def _list(cli_ctx, show_all_plugins):
             click.echo()
 
 
-@cli.command(short_help="Create a new keyfile account with a random private key")
+@cli.command(short_help="Create a new keyfile account with a random mnemonic seed phrase")
+@click.option(
+    "--word-count",
+    "word_count",
+    help="Number of words to use to generate seed phrase",
+    default=12,
+    show_default=True,
+)
+@click.option(
+    "--hdpath",
+    "custom_hd_path",
+    help="Specify an HD path for deriving seed phrase",
+    default=ETHEREUM_DEFAULT_PATH,
+    show_default=True,
+)
 @non_existing_alias_argument()
 @ape_cli_context()
-def generate(cli_ctx, alias):
+def generate(cli_ctx, alias, word_count, custom_hd_path):
     path = _get_container().data_folder.joinpath(f"{alias}.json")
-    extra_entropy = click.prompt(
-        "Add extra entropy for key generation...",
-        hide_input=True,
+    EthAccount.enable_unaudited_hdwallet_features()
+    account, mnemonic = EthAccount.create_with_mnemonic(
+        num_words=word_count, account_path=custom_hd_path
     )
-
-    account = EthAccount.create(extra_entropy)
+    cli_ctx.logger.success(f"Newly generated mnemonic is: {mnemonic}")
     passphrase = click.prompt(
         "Create Passphrase",
```

Comment:
This might make it clearer :thinking: 
```suggestion
        "Create Passphrase to encrypt account",
```

Also maybe update `accounts import` too